src/docs/sphinx/IntelGPU.rst

```diff
@@ -103,6 +103,22 @@ Variorum provides an API to report instantaneous clock speed of the Intel GPU's
 execusion unit in MHz and integer precision. It leverages the
 ``apmidg_readfreq()`` APMIDG API to report the instantaneous clock speed.
 
+*******************
+ Control Interface
+*******************
+
+The Intel Discrete GPU port of Variorum leverages the device-level control APIs
+provided by APMIDG. Variorum implements the following device control APIs using
+the corresponding APMIDG APIs.
+
+Power control
+=============
+
+Variorum provides an API to cap the power usage of an Intel Discrete GPU device.
+To set the power cap, Variorum uses the ``apmidg_setpwrlim()`` API of APMIDG
+which takes as input the GPU device ID, the power domain ID and the power cap in
+milliwatts.
+
```

Comment:
Very minor comment, feel free to ignore. 

The text in this rst file seems a little repetitive: 
In the the "Power control" subsection, you can maybe remove the first sentence ("Variorum provides...device"), and edit the second sentence to say "In Variorum's GPU power capping API, Variorum uses the ``apmidg....`` API, which takes as input...milliwatts".